dq-server/src/routes/games.js

```diff
@@ -22,6 +22,20 @@ router.get('/:id', auth, asyncCatch(async (req, res) => {
 	return res.send(game);
 }));
 
+router.put('/:id', auth, asyncCatch(async (req, res) => {
+	let result = updateGameSchema.validate(req.body);
+	if (result.error) {
+		res.status(400).send(result.error.details[0].message);
+		return;
+	}
+	try {
+		result = await gamesController.getGameAndUpdate(req.params.id, req.body);		
+	} catch (error) {
+		res.status(404).send(error.message);
```

Comment:
will `getGameAndUpdate` throw error when not found? or just return `none`?

Reply:
![image](https://user-images.githubusercontent.com/17834635/104762182-b02b0700-5742-11eb-897a-b47967dfd0a2.png)

Reply:
That is because what you sent as param is not a valid ObjectID. If you were to send a valid objectID that doesn't correspond to a game, it will send a 200 with an empty response body